routing-lab/src/Component/MultCats.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import axios from 'axios'
+import Image from './Image'
+
+
+class MultCats extends React.Component{
+    constructor(props){
+        super(props)
+        state ={ apiKey:"1e37d765-8afa-40d5-b975-78fdd644a659", catPics: [], numberOfCats:1}
```

Comment:
You really don't want to upload the API key with your repo, this is like giving everyone keys to your house. You want to create a special file and put the key there and then add that file to your .gitignore